src/pages/index.tsx

```diff
@@ -1,75 +1,88 @@
-import { Button, Card, Text, Spacer, useTheme } from '@geist-ui/core';
+import {Button, Text, Spacer, useTheme} from '@geist-ui/core';
 import NextLink from 'next/link';
 import dynamicWidth from '@/lib/dynamic-width';
-import { useState } from 'react';
+import {useState} from 'react';
```

Comment:
Please add space after { and before }
Also, you might want to install a linter.